src/main/java/org/example/commands/DeleteDependencyCommand.java

```diff
@@ -2,39 +2,56 @@
 
 import java.util.ArrayList;
 import java.util.Map;
-import java.util.Scanner;
 import org.example.exceptions.InvalidArgument;
 import org.example.model.Node;
+import org.example.services.ConsoleInputServiceInterface;
 
 public class DeleteDependencyCommand implements CommandInterface {
+  private final ConsoleInputServiceInterface consoleInputService;
```

Comment:
u have removed this private final in future prs. What is the reason?

Reply:
There were some issues with the scanner , added private final again